AFQ/segmentation.py

```diff
@@ -642,6 +648,33 @@ def segment_afq(self, tg=None):
                 self.fiber_groups[bundle] = select_sl
         return self.fiber_groups
 
+    def move_streamlines(self, tg=None, slr_algo='linear'):
+        """Streamline-based registration of a whole-brain tractogram to
+        the MNI whole-brain atlas.
+
+        registration_algo : str
+            "linear" or "syn"
+        """
+        if tg is None:
+            tg = self.tg
+        else:
+            self.tg = tg
+
+        if slr_algo == "linear":
+            self.logger.info("Registering tractogram with linear SLR")
+            atlas = self.bundle_dict['whole_brain']
+            self.moved_sl, _, _, _ = whole_brain_slr(
+                atlas, self.tg.streamlines, x0='affine', verbose=False,
+                progressive=self.progressive,
+                greater_than=self.greater_than,
+                rm_small_clusters=self.rm_small_clusters,
+                rng=self.rng)
+        elif slr_algo == "syn":
```

Comment:
following the convention in the api.AFQ object, the variable name should be reg_algo and the values should be "slr" or "syn". Also, there should be a way for the api to pass in its own mapping which it already created.

Reply:
We might want to just remove SLR from the segmentation class and ask for a mapping. Just like syn, if the user wants to use slr, they should use: https://github.com/yeatmanlab/pyAFQ/blob/9dda145ee06314a45862c07a0137c523276dd5f2/AFQ/registration.py#L476
This reduces redundant code and makes it so all mapping have the same interface.